src/mscorlib/shared/System/Memory.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace System
 {
+    [IsReadOnly]
```

Comment:
Could you please wait with this until we get the C# compiler upgrade in CoreCLR repo? All these places will need to touched as part of the upgrade to the new syntax. We are having troubles getting the C# compiler upgrade through because there are too many places that have these annotations already, and they break for various reasons.

Reply:
Sure. Closing the PR.